.eslintrc.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I think you have to instate the ava override in the `spec/` directory, like how https://github.com/mixmaxhq/salesforce/pull/87/files#diff-16f3267d579b1850bd1153130fd8286d works